src/tab/Tab.tsx

```diff
@@ -16,20 +16,26 @@ export interface TabProps {
   items: TabItem[];
   children: React.ReactNode[];
   testid?: string;
-  initialActiveTabIndex?: number;
+  activeTabIndex?: number;
```

Comment:
There should be two props: `activeTabIndex` and `initialActiveTabIndex`. If `activeTabIndex` is provided, then that should mean Tab component shouldn't handle the state itself but to delegate state management to its parent (ie become a controlled component). `initialActiveTabIndex` is there to set the active initial tab when  `activeTabIndex` was not provided.

See https://reactjs.org/docs/uncontrolled-components.html